docs/static/releasenotes.asciidoc

```diff
@@ -18,6 +19,54 @@ This section summarizes the changes in the following releases:
 * <<logstash-7-0-0-alpha2,Logstash 7.0.0-alpha2>>
 * <<logstash-7-0-0-alpha1,Logstash 7.0.0-alpha1>>
 
+[[logstash-7-5-0]]
+=== Logstash 7.5.0 Release Notes
+
+* Give multiple pipelines all the settings https://github.com/elastic/logstash/pull/11076[#11076]
+* Support substitutions in pipelines.yml file https://github.com/elastic/logstash/pull/11081[#11081]
+* Provide feature/log appender per pipeline https://github.com/elastic/logstash/pull/11108[#11108]
+* Use 2048 bits key in OpenSSL socket specs https://github.com/elastic/logstash/pull/11115[#11115]
+* Add origins of pipeline configurations https://github.com/elastic/logstash/pull/11123[#11123]
+* Update .ruby-version to jruby-9.2.8.0 https://github.com/elastic/logstash/pull/11198[#11198]
```

Comment:
We can remove this one. It's a tweak in the environment setup for testing, not relevant for the release.

```suggestion
```